src/modules/colorPicker/ColorPickerUI/Bootstrapper.cs

```diff
@@ -11,6 +11,7 @@ public static class Bootstrapper
     {
         public static CompositionContainer Container { get; private set; }
 
+        [System.Diagnostics.CodeAnalysis.SuppressMessage("Reliability", "CA2000:Dispose objects before losing scope", Justification = "This is properly disposed of in MainWindow.Xaml.cs")]
```

Comment:
@crutkas .  I'm a bit confused by this.  The error to me looks like we should be disposing AssemblyCatalog, but the documentation for [AssemblyCatalog ](https://docs.microsoft.com/en-us/dotnet/api/system.componentmodel.composition.hosting.assemblycatalog?view=dotnet-plat-ext-3.1)doesn't dispose it.  It might be better to have a private member `_catalog` and Dispose it in the Dispose method, instead of supressing this error?

Reply:
if you wrap anything in that statement with a using, the behavior will fail.

var mouseInfoProvider = Bootstrapper.Container.GetExportedValue<IMouseInfoProvider>(); for instance.

We could refactor and move this entirly into app or main window.  @martinchrzan thoughts?

Reply:
Sorry I was suggesting that we Dispose the Assembly Catalog in BootStrapper.Dispose(), but I'll leave that up to you as I don't know what the impact of that is, and any resources (if any) that weren't disposed would likely be killed with the process.

Reply:
I would be surprised it wasn't disposed by disposing the container's dispose.